shaptools/hana.py

```diff
@@ -195,14 +202,26 @@ def install(cls, software_path, conf_file, root_user, password, remote_host=None
             conf_file (str): Path to the configuration file
             root_user (str): Root user name
             password (str): Root user password
+            hdb_pwd_file (str, opt): Path to the XML password file
             remote_host (str, opt): Remote host where the command will be executed
         """
         # TODO: mount partition if needed
         # TODO: do some integrity check stuff
+
+        if not os.path.isfile(conf_file):
+            raise FileDoesNotExist('The configuration file does not exist')
+        if hdb_pwd_file is not None and not os.path.isfile(hdb_pwd_file):
```

Comment:
It could just `if hdb_pwd_file and not os.path.isfile(hdb_pwd_file):` but it's OK the current one too